.gitignore

```diff
@@ -10,3 +10,4 @@
 !/.yarn/releases
 !/.yarn/sdks
 !/.yarn/versions
+/utils/e2e-tests/ts/node_modules
```

Comment:
Should not be required, as we are using a subgle global `node_modules` dir at the root level...

Reply:
Solved this in a proper way at #1401